assets/js/console.js

```diff
@@ -253,8 +254,7 @@ dropzone.addEventListener(
   e => {
     e.preventDefault()
     e.stopPropagation()
-
-    ;[...e.dataTransfer.items].forEach((item, i) => {
+    ;[...e.dataTransfer.items].forEach(item => {
```

Comment:
This pains me 😅 There's no need to spread here, so we can just:
```suggestion
    e.dataTransfer.items.forEach(item => {
```